foundations/installations/installing_node.md

```diff
@@ -38,18 +42,51 @@ You can verify `nvm` is installed by running the command:
 
 
 ~~~BASH
-command -v nvm
+command nvm -v
```

Comment:
This should remain as `command -v nvm`. The `-v` is a flag of the `command` command.

```
mfrank ~ command nvm -v
zsh: command not found: nvm
mfrank ~ command -v nvm
nvm
```